appinfo/routes.php

```diff
@@ -46,110 +46,111 @@
 	],
 	'ocs' => [
 		[
-			'name' => 'api#getRooms',
-			'url' => '/api/{apiVersion}/room',
-			'verb' => 'GET',
-			'requirements' => ['apiVersion' => 'v1'],
-		],
-		[
-			'name' => 'api#makePublic',
-			'url' => '/api/{apiVersion}/room/public',
-			'verb' => 'POST',
-			'requirements' => ['apiVersion' => 'v1'],
-		],
-		[
-			'name' => 'api#makePrivate',
-			'url' => '/api/{apiVersion}/room/public',
-			'verb' => 'DELETE',
-			'requirements' => ['apiVersion' => 'v1'],
-		],
-		[
-			'name' => 'api#getRoom',
-			'url' => '/api/{apiVersion}/room/{token}',
+			'name' => 'Call#getPeersForCall',
+			'url' => '/api/{apiVersion}/room/{token}/peers',
 			'verb' => 'GET',
 			'requirements' => [
 				'apiVersion' => 'v1',
 				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
 		[
-			'name' => 'api#renameRoom',
-			'url' => '/api/{apiVersion}/room/{roomId}',
-			'verb' => 'PUT',
+			'name' => 'Call#joinCall',
+			'url' => '/api/{apiVersion}/room/{token}/join',
+			'verb' => 'POST',
 			'requirements' => [
 				'apiVersion' => 'v1',
-				'roomId' => '\d+'
+				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
 		[
-			'name' => 'api#addParticipantToRoom',
-			'url' => '/api/{apiVersion}/room/{roomId}',
+			'name' => 'Call#pingCall',
+			'url' => '/api/{apiVersion}/ping',
 			'verb' => 'POST',
 			'requirements' => [
 				'apiVersion' => 'v1',
-				'roomId' => '\d+'
+				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
 		[
-			'name' => 'api#leaveRoom',
-			'url' => '/api/{apiVersion}/room/{roomId}',
+			'name' => 'Call#leaveCall',
+			'url' => '/api/{apiVersion}/leave',
```

Comment:
The problem is, this frees you up from all calls, in case your internet connection was interrupted you could not join another call then, because your session is still in this one.